### ISSUE TITLE
The <redoc> Options object and Theme object docs look out of date

### ISSUE THREAD

**Issue by ajrice6713** [Redocly/redoc#1607]
Starting here https://github.com/Redocly/redoc/blob/master/README.md#redoc-options-object

Ive noticed a few deprecated theme/object options that seem to be mis-documented. `menu` in the Theme object seems to be no longer valid and replaced by `sidebar`

Would it be possible to get these updated for easier integration?

**Comment by ajrice6713** [comment]
Awesome - thanks for the quick reply! 

I guess this issue can be closed if the link you provided covers everything!

**Comment by skadinna** [comment]
Yes, the linked docs cover all config options for Redoc. There's also [this page with theming options](https://redoc.ly/docs/api-reference-docs/configuration/theming/). It doesn't have detailed descriptions yet, but the schema structure is up-to-date, and it indicates which options are supported in Redoc. So perhaps that can be useful, too.

I believe we'll see some serious improvements to our Redoc documentation soon enough. ;) Thanks for your patience!

**Comment by gbokoua** [comment]
[cid:22e0863e-b7b3-4205-a0b3-16c73d8cbf34]
________________________________
